### PR TITLE
Add support for TraceMultiPropagator

### LINKED ISSUE #637: Add support for TraceMultiPropagator
opentelemtry 0.6 comes with [TraceMultiPropagator](https://github.com/open-telemetry/opentelemetry-java/blob/master/extensions/trace_propagators/src/main/java/io/opentelemetry/extensions/trace/propagation/TraceMultiPropagator.java#L60)

current [PropagatorsInitializer](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/master/agent-tooling/src/main/java/io/opentelemetry/auto/tooling/PropagatorsInitializer.java#L30) supports only 'tracecontext' or 'b3'.

### PR DESCRIPTION
**Pull request by malafeev**
fix #637 
cc @carlosalberto

### CODE REVIEW

File: agent-tooling/src/main/java/io/opentelemetry/auto/tooling/PropagatorsInitializer.java [hunk 1]
```diff
@@ -60,23 +63,26 @@ public static void initializePropagators(List<String> propagators) {
 
     DefaultContextPropagators.Builder propagatorsBuilder = DefaultContextPropagators.builder();
 
-    boolean addedPropagator = false;
+    List<HttpTextFormat> textPropagators = new ArrayList<>();
```
Comment (iNikem) [inline agent-tooling/src/main/java/io/opentelemetry/auto/tooling/PropagatorsInitializer.java:66]: Nitpicking: `new ArrayList<>(propagators.size())`

Comment (malafeev) [reply agent-tooling/src/main/java/io/opentelemetry/auto/tooling/PropagatorsInitializer.java]: done

**Comment by iNikem** [comment]
A! We forgot to document this behaviour! @trask @malafeev

**Comment by trask** [comment]
:thumbsup: I will add the doc update to #830

**Comment by trask** [comment]
> 👍 I will add the doc update to #830

done

**Pull request merged**